src/routes/safe/components/Transactions/helpers/Simulation/SimulationResult.tsx

```diff
@@ -0,0 +1,84 @@
+import { Icon, Link, Text } from '@gnosis.pm/safe-react-components'
+import { Box, IconButton } from '@material-ui/core'
+import { Close } from '@material-ui/icons'
+import { FETCH_STATUS } from 'src/utils/requests'
+import { TenderlySimulation } from './types'
+
+type SimulationResultProps = {
+  simulationRequestStatus: string
+  simulation?: TenderlySimulation
+  simulationLink?: string
+  simulationError?: string
+  onClose: () => void
+}
+
+export const SimulationResult = ({
+  simulationRequestStatus,
+  simulation,
+  simulationLink,
+  simulationError,
+  onClose,
+}: SimulationResultProps): React.ReactElement => {
+  const isError = !!simulationError || !simulation?.simulation.status
+  return (
+    <>
+      {(simulationRequestStatus === FETCH_STATUS.SUCCESS || Boolean(simulationError)) && (
```

Comment:
Should we display anything if `simulation` is `undefined`? I would also suggest splitting up the conditions because there is some overlap i.e. if there is a `simulationError` do we even need to check `simulation.status`?

Reply:
`simulationError`is only set if there is an unexpected error. For instance the proxy down, etc. It's an error on http level. In this case the `simulation` object will be undefined.
If a simulation fails the `simulation.status`which is a boolean will be `false` if the transaction simulation was successful it will return `true`

I will rename `simulationError` to `requestError` to make this difference more clear.